src/ProjectService.Validation/EditTaskPropertyValidator.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Enums;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models;
+using LT.DigitalOffice.ProjectService.Validation.Interfaces;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+
+namespace LT.DigitalOffice.ProjectService.Validation
+{
+  public class EditTaskPropertyValidator : AbstractValidator<JsonPatchDocument<TaskProperty>>, IEditTaskPropertyValidator
+  {
+    private readonly ITaskPropertyRepository _taskPropertyRepository;
+    private readonly IUserRepository _userRepository;
+
+    public EditTaskPropertyValidator(
+      ITaskPropertyRepository taskPropertyRepository,
+      IUserRepository userRepository)
+    {
+      _taskPropertyRepository = taskPropertyRepository;
+      _userRepository = userRepository;
+
+      RuleForEach(x => x.Operations)
+        .Custom(HandleInternalPropertyValidation);
+    }
+
+    private void HandleInternalPropertyValidation(Operation<TaskProperty> requestedOperation, CustomContext context)
+    {
+      #region local functions
```

Comment:
remove this region and use kernel 41 version

Reply:
done